quiz/views.py

```diff
@@ -0,0 +1,143 @@
+from collections import defaultdict
+from django.shortcuts import get_object_or_404
+from django.utils.translation import gettext_lazy as _
+from rest_framework import viewsets, mixins, response, exceptions, permissions
+from .models import Question, Quiz, Answer, Category, Score
+from .serializers import (QuestionSerializer, QuestionListSerializer,
+                          QuizListSerializer, QuizSerializer,
+                          AnswerAdminSerializer, CategorySerializer,
+                          QuestionCreateSerializer, GameCreateSerializer,
+                          QuizResultSerializer)
```

Comment:
@diogor Aqui o mesmo caso, você poderia evitar fazendo o import por módulo `from . import serializers`

Reply:
Sim, vou fazer assim quando for o caso de usar vários imports, realmente fica bem mais limpo.